src/suites/cts/command_buffer/render/rendering.spec.ts

```diff
@@ -18,14 +20,15 @@ g.test('fullscreen quad', async t => {
 
   const vertexModule = t.makeShaderModule(
     'v',
-    `#version 450
+    `#version 310 es
     void main() {
       const vec2 pos[3] = vec2[3](
           vec2(-1.f, -3.f), vec2(3.f, 1.f), vec2(-1.f, 1.f));
       gl_Position = vec4(pos[gl_VertexIndex], 0.f, 1.f);
     }
   `
   );
+  // TODO: Fix glslang build so '310 es' can be used here (crashes currently)
   const fragmentModule = t.makeShaderModule(
     'f',
     `#version 450
```

Comment:
It works with this? Or does the test just fail?

Reply:
(It works with `#version 450`?)

Reply:
yup, it works with 450. Updated comment.